backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -418,6 +419,72 @@ def notify_created_account(
             message=message
         )
 
+    def notify_reset_password(
+            self,
+            user: User,
+            reset_password_token: str,
+    ) -> None:
+        """
+        Reset password link for user
+        :param user: user to notify
+        :param reset_password_token: token for resetting password
+        """
+        logger.debug(self, 'user: {}'.format(user.user_id))
+        logger.info(self, 'Sending asynchronous email to 1 user ({0})'.format(
+            user.email,
+        ))
+        translator = Translator(self.config, default_lang=user.lang)
+        async_email_sender = EmailSender(
+            self.config,
+            self._smtp_config,
+            self.config.EMAIL_NOTIFICATION_ACTIVATED
+        )
+        subject = self.config.EMAIL_NOTIFICATION_RESET_PASSWORD_SUBJECT.replace(
+            EST.WEBSITE_TITLE,
+            self.config.WEBSITE_TITLE.__str__()
+        )
+        message = MIMEMultipart('alternative')
+        message['Subject'] = subject
+        message['From'] = self._get_sender()
+        message['To'] = formataddr((user.get_display_name(), user.email))
+
+        text_template_file_path = self.config.EMAIL_NOTIFICATION_RESET_PASSWORD_TEMPLATE_TEXT  # nopep8
+        html_template_file_path = self.config.EMAIL_NOTIFICATION_RESET_PASSWORD_TEMPLATE_HTML  # nopep8
+        # TODO - G.M - 2018-08-17 - Generate token
+        context = {
+            'user': user,
+            # TODO - G.M - 11-06-2018 - [emailTemplateURL] correct value for logo_url  # nopep8
+            'logo_url': get_email_logo_frontend_url(self.config),
+            # TODO - G.M - 11-06-2018 - [emailTemplateURL] correct value for login_url  # nopep8
```

Comment:
What do you use as value for `login_url` if it is a TODO ?

Reply:
value are correct now, todo are deprecated. logo_url will probably be changed but not now.